cpp/test/test.cpp

```diff
@@ -0,0 +1,77 @@
+#include "acutest.h"
+#include "UserId.hpp"
+#include <cstdlib>
+#include <iostream>
+#include <boost/optional.hpp>
+#include "UserId_impl.hpp"
+#include "DocumentListMeta.hpp"
+#include "DocumentMetadataResult.hpp"
+#include "IronOxide_fwd.hpp"
+#include "IronOxide_impl.hpp"
+#include "DeviceContext_impl.hpp"
+#include "DocumentEncryptResult_impl.hpp"
+using namespace sdk;
+
+template <class T>
+T unwrap(std::variant<T, RustString> value)
+{
+    if (value.index() == 0)
+    {
+        return std::get<T>(std::move(value));
+    }
+    else
+    {
+        std::cout << "Got fatal error: " << std::get<RustString>(value).to_string_view() << "\n";
```

Comment:
This is our equivalent of panic, I guess?

Reply:
It's a hack in our tests for now. We probably shouldn't exit, but when we beef up the tests we can fix that.